src/components/Common/TabGroup/styles.ts

```diff
@@ -0,0 +1,32 @@
+import type { ThemeUICSSObject } from 'theme-ui';
+
+const tabActive: ThemeUICSSObject = {

```

Comment:
Why sometimes use Upper case on first case, sometimes use lower case?

Reply:
uppercase for react component and constant value (such as action name)
lower case for normal object value, this time is for css object